files/image_config/caclmgrd/caclmgrd

```diff
@@ -88,24 +92,48 @@ class ControlPlaneAclManager(object):
     }
 
     def __init__(self):
-        # Open a handle to the Config database
-        self.config_db = ConfigDBConnector()
-        self.config_db.connect()
+        SonicDBConfig.load_sonic_global_db_config()
+        self.config_db_map = {}
+        self.iptables_cmd_ns_prefix = {}
+        self.config_db_map[''] = ConfigDBConnector(use_unix_socket_path=True, namespace='')
+        self.config_db_map[''].connect()
+        self.iptables_cmd_ns_prefix[''] = ""
+        self.docker_bridge_ip = self.get_docker_ip_address(self.iptables_cmd_ns_prefix[''], '')
+        self.namespace_docker_mgmt_ip = {}
+        namespaces = device_info.get_all_namespaces()
+        for front_asic_namespaces in namespaces['front_ns']:
```

Comment:
Variable name should be singular. I.e., `front_asic_namespace`

Reply:
@jleveque Done